src/Components/Board/Board.js

```diff
@@ -0,0 +1,19 @@
+import React, { useContext } from 'react';
+import { UserContext } from '../../Context/GameContext.js';
+import Box from '../Box/box.js';
+import './Board.css';
+
+export default function Board() {
+  const { boxes, turn, winnerMessage } = useContext(UserContext);
+  return (
+    <>
+      <h1>{winnerMessage}</h1>
+      <h2>{turn}s Turn</h2>
+      <div className="game-container">
+        {boxes.map((box) => {
+          return <Box key={box.space} space={box.space} content={box.content} />;
```

Comment:
if you use ```(``` on line 13 instead of ```{```, everything on line 14 will be implicitly returned so you wouldn't have to name it